site/public/js/sql-toolbox.js

```diff
@@ -73,3 +74,33 @@ async function runSqlQuery() {
         console.error(exc);
     }
 }
+
+async function downloadSqlResult(){
+    const results = document.getElementById('query-results');
+    let i;
+    let j;
+    let csv = '';
+    //Add headers to CSV string
+    for (i = 1; i < results.children.item(0).children.item(0).children.length; i++){
+        csv += `"${results.children.item(0).children.item(0).children.item(i).textContent}",`;
+    }
```

Comment:
const header_elements = results.children.item(0).children.item(0).children;
    for (i = 1; i < header_elements.length; i++){
        csv += `"${header_elements.item(i).textContent}",`;
    }